### PR TITLE
Issue 1264 refactor affine

### LINKED ISSUE #1264: Move/rename makeAffineTransform and decomposeAffineTransform to transform.js
- `makeAffineTransform()` -> `decomposedToTransform()`
- `decomposeAffineTransform()` -> `decomposedFromTransform()`

We don't have a `DecomposedTransform` class like fonttools has, but at least this would make those functions more similar to `DecomposedTransform.fromTransform()` and `DecomposedTransform().toTransform()`.

This depends on #1247.

### PR DESCRIPTION
**Pull request by ollimeier**
Fixes #1264

### CODE REVIEW

**Comment by ollimeier** [comment]
@justvanrossum Even though I was not able to fix it, yet – I have some good findings: With the help of chai-almost I was able to reduce the number of fails and find a pattern. The big differences are always related to the 'yx'-values. I will investigate more tomorrow.

**Comment by justvanrossum** [comment]
I committed two things:
- a minor fix that was done to the fonttools implementation (`>=` vs `>` in two places)
- Compute `delta` after the `sign()` call, this matches the fonttools implementation, and makes the tests pass

File: src/fontra/client/core/transform.js [hunk 1]
```diff
@@ -167,6 +167,11 @@ export class Transform {
     return new this.constructor(xx, xy, yx, yy, dx, dy);
   }
 
+  toDecomposed() {
+    //Decompose into a DecomposedTransform.
```
Comment (justvanrossum) [inline src/fontra/client/core/transform.js:171]:
```suggestion
    // Decompose into a DecomposedTransform.
```

**Review by justvanrossum — APPROVED** [review]
One nit, otherwise good to go!

**Pull request merged**